src/app/user/user.service.ts

```diff
@@ -132,12 +132,9 @@ export class UserService {
 	}
 
 	getUserPermissions(){
```

Comment:
Need to enforce those permissions server sided.

Reply:
will use this now to hide options to delete / edit songs that u dont belong to. it wont be server side cause server shouldnt know which action is triggered. ill do it in angular